README_DEVELOPERS.md

```diff
@@ -68,6 +73,26 @@ drone exec --matrix TARGET=test
 
 ### In a Cloud CI
 
-We provide configuration YAML files for [Travis CI](https://travis-ci.org/) ([.travis.yml](.travis.yml)) and [Shippable](https://app.shippable.com/) ([shippable.yml](shippable.yml)). Enable one of them for your forked repo (we recommend Travis). 
+We provide configuration YAML files for [Travis CI](https://travis-ci.org/) ([.travis.yml](.travis.yml)) and [Shippable](https://app.shippable.com/) ([shippable.yml](shippable.yml)). Enable one of them for your forked repo (we recommend Travis).
 
 Please make sure that each pull request passes all automatic checks before submitting it.
+
+## Commit Message Guidelines
+
+We have a few rules of formatting commit messages and pull request titles:
+
+- Use the imperative mood in the subject line
+- Capitalize the subject line
+- Do not end the subject line with a period
+- Limit the subject line to 50 characters
+- Separate the subject from the body with a blank line
+- Wrap the body at 72 characters
+- Use the body to explain what and why vs how
```

Comment:
Based on https://chris.beams.io/posts/git-commit/?
Please, add emphases and the dot.

Reply:
>Please, add emphases and the dot.

Do we really need it?

Reply:
Sure. They are used for a reason in the source.

Reply:
I'm not sure, that I understand what exactly you mean...